<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>atualizarFuncionario</class>
 <widget class="QMainWindow" name="atualizarFuncionario">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciamento de Funcion√°rios</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>minilogo.png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #EDF2F4;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtId">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>205</y>
      <width>104</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDataNasc">
    <property name="geometry">
     <rect>
      <x>574</x>
      <y>297</y>
      <width>142</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>AA/MM/DD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefone">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>377</y>
      <width>220</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
    <property name="placeholderText">
     <string>(00) 00000-0000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>377</y>
      <width>220</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCpf">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>297</y>
      <width>142</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
    <property name="placeholderText">
     <string>000.000.000-00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRg">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>297</y>
      <width>142</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
    <property name="placeholderText">
     <string>00.000.000-0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNome">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>205</y>
      <width>228</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">box-sizing: border-box;

position: absolute;
width: 352px;
height: 24px;
left: 364px;
top: 205px;

background: #EDF2F4;
border: 1px solid #8D99AE;
border-radius: 8px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmar">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>448</y>
      <width>142</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

box-sizing: border-box;

position: absolute;
width: 141px;
height: 64px;
left: 413px;
top: 448px;

background: #FFFFFF;
border: 2px solid #07D977;
border-radius: 16px;

position: absolute;
left: 2.84%;
right: 2.84%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #07D977;

}

QPushButton:hover{

box-sizing: border-box;

position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #07D977;
border: 2px solid #07D977;
border-radius: 16px;

position: absolute;
left: 0.7%;
right: 0.7%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #FFFFFF;

}
</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExcluir">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>462</y>
      <width>142</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
position: absolute;
width: 141px;
height: 64px;
left: 24px;
top: 266px;

border-radius: 16px;

position: absolute;
left: 2.84%;
right: 2.84%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #0C033C;

}

QPushButton:hover{
position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #0C033C;
border-radius: 16px;

position: absolute;
left: 0.7%;
right: 0.7%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #FFFFFF;

}
</string>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnverTodos">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>364</y>
      <width>142</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
position: absolute;
width: 141px;
height: 64px;
left: 24px;
top: 266px;

border-radius: 16px;

position: absolute;
left: 2.84%;
right: 2.84%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #0C033C;

}

QPushButton:hover{
position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #0C033C;
border-radius: 16px;

position: absolute;
left: 0.7%;
right: 0.7%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #FFFFFF;

}
</string>
    </property>
    <property name="text">
     <string>Ver Todos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtualizar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>266</y>
      <width>142</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #0C033C;
border-radius: 16px;

position: absolute;
left: 0.7%;
right: 0.7%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #FFFFFF;
</string>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInserir">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>168</y>
      <width>142</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
position: absolute;
width: 141px;
height: 64px;
left: 24px;
top: 266px;

border-radius: 16px;

position: absolute;
left: 2.84%;
right: 2.84%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #0C033C;

}

QPushButton:hover{
position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #0C033C;
border-radius: 16px;

position: absolute;
left: 0.7%;
right: 0.7%;
top: 31.25%;
bottom: 31.25%;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 24px;
/* identical to box height */

text-align: center;

color: #FFFFFF;

}
</string>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>94</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 800px;
height: 94px;
left: 0px;
top: 0px;

background: #0C033C;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>32</y>
       <width>392</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">position: absolute;
width: 344px;
height: 36px;
left: 295px;
top: 32px;

font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 24px;
line-height: 36px;
/* identical to box height */

background: #0C033C;
color: #FFFFFF;
</string>
     </property>
     <property name="text">
      <string>Gerenciamento de Funcion√°rios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>32</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
image: url(:/logo/1x/arrow.png);

background: #0C033C;
position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

border-radius: 16px;
}

QPushButton:hover{

image: url(:/logo/1x/arrowhover.png);

background: #0C033C;

position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;
border-radius: 16px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <selectedon>arrow.png</selectedon>
      </iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>20</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>iconelogo.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>142</y>
      <width>554</width>
      <height>410</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 554px;
height: 410px;
left: 206px;
top: 142px;

background: #FFFFFF;
box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.25);
border-radius: 16px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>188</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>360</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>280</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>CPF:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>280</y>
      <width>132</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>Data de Nascimento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>360</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>Celular:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>280</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>RG:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>188</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
font-family: 'Poppins';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 18px;
/* identical to box height */


color: #000000;
</string>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>iconedit.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>txtId</zorder>
   <zorder>txtDataNasc</zorder>
   <zorder>txtTelefone</zorder>
   <zorder>txtEmail</zorder>
   <zorder>txtCpf</zorder>
   <zorder>txtRg</zorder>
   <zorder>txtNome</zorder>
   <zorder>btnConfirmar</zorder>
   <zorder>btnExcluir</zorder>
   <zorder>btnverTodos</zorder>
   <zorder>btnAtualizar</zorder>
   <zorder>btnInserir</zorder>
   <zorder>frame</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_12</zorder>
  </widget>
 </widget>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
